README.md

```diff
@@ -1,24 +1,136 @@
 # Sinatra Fwitter 2 -  Forms
 
-## Outline
+## Objectives
 
-1. Create a new file, `tweet.erb`, to render a form. The form should have inputs for username and status. 
-2. Create a route in the controller to render the `tweet.erb` template. Create another route for when the form is submitted. 
-3. In the `post` route, create a new instance of our Tweet class based on params. Redirect to the '/' route (instruct users to use `rackup` instead of `shotgun` to run their server, or they won't see the newly created tweet.
+1. Understand the difference between a "get" and a "post" request. 
+2. Create a form with a method and action.
+3. Use data submitted through a form in `params` to instantiate new instances of objects
 
-## Objectives
+## Overview
 
-1. Objective 1
-2. Objective 2
+Last time in Fwitter, we created the basic layout of our Model-View-Controller file structure. Today, let's add a form so that users can dynamically create new tweets. 
 
 ## Instructions
 
-Create make the tests pass Heroku lab. API Nokogiri guest speaker belongs_to RESTful binder.ply. Url Rails slack it to me. Asset pipeline Feelings Friday puts "woof" API open source.
+### Setup
+
+Fork and clone this repository to get started. You'll notice in our index page we've added a link to the [Bootstrap](http://getbootstrap.com/) stylesheet - an great CSS library for incorporating some layout and structure very quickly. 
+
+*Note: For this lab, make sure to start your server using `rackup` instead of `shotgun`. This means that you'll have to restart your server every time you update your code, but otherwise your tweets won't display properly on the index page.*
+
+### Creating a Form
+
+The first thing we'll need for our users to be able to post their own tweets is a form. In the `views` directory, create a new file called `tweet.erb`. We'll also need to add a route to our application controller which will render the form. Make a new route for `get '/tweet'` which renders our new file. 
+
+```ruby
+	get '/tweet' do
+	  erb :tweet
+	end
+```
+
+Let's build out a form in our `tweet.erb` file. It should have an action of `/tweet` and a method of `post`. 
+
+```ERB
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fwitter</title>
+  </head>
+  <body> 
+      <h1>Add a tweet</h1>
+      <form action="/tweet" method="POST">
+
+      </form>
+
+  </body>
+</html>
+```
+Our form needs two inputs - one for `username` and one for status. We'll use a type of `text` for each one, as well as `name`. The `name` attribute is very important - this will be the key in the `params` hash when the user makes their post request. We can make the `name` attribute whatever we want, but `username` and `status` make sense. Finally, we also need a Submit button so users can submit the form.
```

Comment:
back-ticks around status are missing.

Our form needs two inputs - one for `username` and one for `status`.